mathics/builtin/arithmetic.py

```diff
@@ -732,35 +732,35 @@ class Real_(Builtin):
 
     ## Formatting tests
     #> 1. * 10^6
```

Comment:
Of course, in the long term we'd like to reduce `#>` kinds of tests. Eventually all of this homegrown kind of formatting and testing should go away.  The `#>` part, thanks to @mmatera, we can replace. 

But if this is too onerous, it can be done later.

Reply:
> Of course, in the long term we'd like to reduce `#>` kinds of tests. Eventually all of this homegrown kind of formatting and testing should go away. The `#>` part, thanks to @mmatera, we can replace.
> 
> But if this is too onerous, it can be done later.

@rocky, I can start to do that, but probably in a different PR.